cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -191,6 +191,8 @@ spec:
 {{- end }}
         - name: ROOK_ENABLE_FLEX_DRIVER
           value: "{{ .Values.enableFlexDriver }}"
+        - name: ROOK_ENABLE_CRASH_CONTROLLER
```

Comment:
Just as a thought, why not have this as an option in the CephCluster CRD?

Reply:
Because the controller starts before the cluster, and the controller only supports a single CephCluster anyway.
We could have it in the Cluster CRD but this would mean starting the controller and then making it return nil.

With the operator oriented approach, the controller is never started.

Reply:
I'm also struggling with this being in the operator. I thought the crash logs would be collected for a single cluster and reported back to that cluster. Per our discussion, if there are multiple clusters, only the first cluster will have crash logs collected. If there are crashes in the other clusters on the same nodes will they be using the same host path? If so, the first cluster would be collecting the logs for all clusters. Or is there a separate path on the host per cluster so the other logs would be ignored? 

In any case, the feature really should support multiple clusters and have a CR setting. In the short term if we need this setting while only one cluster is supported, could we not add the setting to the cluster CR? It would just be ignored for other clusters.